src/status.rs

```diff
@@ -20,12 +23,34 @@ pub trait Conditions {
     fn conditions_mut(&mut self) -> &mut Vec<Condition>;
 }
 
+/// Provides access to the custom resource status to read, write or delete the current_command.
+pub trait HasCurrentCommand {
+    fn current_command(&self) -> Option<CommandRef>;
+    // TODO: setters are non-rusty, is there a better way? Dirkjan?
+    fn set_current_command(&mut self, command: CommandRef);
+    fn clear_current_command(&mut self);
```

Comment:
I think this is never used - we use clear_current_command from command.rs instead..we could move this over to the same `get_patch` pattern we've used for other stuff maybe?

Reply:
It is used in the same method. Not sure what to do here?